routemodel/routes_elevs.py

```diff
@@ -0,0 +1,235 @@
+                                                    'Direction': direction, 'Street': street}, ignore_index = True)
+        except Exception as error:
+            print(f'Error occurred:', error)
+            print(f'Please try again')
+            return False
+    
+        # write dataframe to csv
+        route_df.to_csv(ROUTES_FILE)
+        print(f'Routes data written to CSV file.')
+        return True
+
+    def parse_elevation_data(self, response):
+        """
+        Parsing through Elevations API call response.
+        @param response: Requests.response object from API call
+        @return: boolean
+        """
+        # convert response into dict
+        elevations = response.json()
+        # parse down to relevant information
+        elevations = elevations['resourceSets'][0]['resources'][0]['elevations']
+
+        # create DataFrame with headers
+        headers = ['Latitude', 'Longitude', 'Elevation']
+        elevation_df = pd.DataFrame(columns=headers)
+
+        # loop through waypoints and viawaypoints
+        # to write coordinates into DataFrame
+        counter = 0
+        for index, waypoint in enumerate(self.waypts):
+            for key in waypoint.keys():
+                elevation_df = elevation_df.append({'Latitude': key,
+                                                    'Longitude': waypoint[key],
+                                                    'Elevation': elevations[counter]}, ignore_index = True)
+                counter += 1
+            try:
+                for viawaypoints in self.viawaypts[index]:
+                    for key in viawaypoints.keys():
+                        elevation_df = elevation_df.append({'Latitude': key,
+                                                            'Longitude': viawaypoints[key],
+                                                            'Elevation': elevations[counter]}, ignore_index = True)
+                    counter += 1
+            except IndexError:
+                pass
+
+
+        # write dataframe into csv and return true
+        elevation_df.to_csv(ELEVATIONS_FILE)
+        print(f'Elevations written to CSV file.')
+        return True
+
+
+if __name__ == "__main__":
+    waypoints = []
+    via_waypoints = []
+    distance_unit = 'km'
+    route_attribute = 'routePath'
+    heights = 'sealevel'
+
+    print(f"Enter up to 25 waypoints.")
+    try:
+        for i in range(25):
+            coordinate = {}
+            latitude = float(input("Type in the latitude: "))
+            longitude = float(input("Type in the longitude: "))
+            coordinate[str(latitude)] = str(longitude)
+            waypoints.append(coordinate)
+    except ValueError:
+        print(f"Waypoints input stopped. Moving on.")
+        pass
+    print(waypoints)
+
+    print(f"Enter up to 10 minor / via waypoints for each waypoint.")
+    for i in waypoints[:-1]:
+        inner_list = []
+        try:
+            for j in range(10):
```

Comment:
minor change here, should store 10 and 25 as variables, instead of them being *magic* numbers

Reply:
Resolved.